packages/editor/src/components/browser/CmsBrowser.tsx

```diff
@@ -63,26 +79,44 @@ const CmsBrowser = ({ value, onChange }: { value: string; onChange: (value: stri
   const [expanded, setExpanded] = useState<ExpandedState>(false as ExpandedState);
   const [globalFilter, setGlobalFilter] = useState('');
   const [rowSelection, setRowSelection] = useState<RowSelectionState>({});
+  const [columnVisibility, setColumnVisibility] = useState<VisibilityState>({ type: false, values: false });
```

Comment:
totally fine. Not sure if you can simply set the visibility on the columndef and then maybe you don't even need this state here, but I'm not sure...

Reply:
I originally wanted to do it this way, but in v8 this is apparently no longer so easily possible via the columndef: https://github.com/TanStack/table/discussions/4106

From my understanding, the solution I have currently implemented (I believe) is intentional and has already been set up to allow for the faster integration of potential radio buttons for filtering.